pkgs/development/compilers/dotnet/combine-packages.nix

```diff
@@ -21,7 +21,7 @@ assert lib.assertMsg ((builtins.length packages) > 0)
       makeWrapper
     ];
     postBuild = ''
-      cp -R ${cli}/{dotnet,LICENSE.txt,nix-support,ThirdPartyNotices.txt} $out/
+      cp -R ${cli}/{dotnet,share,nix-support} $out/
```

Comment:
We can't use symlinks here?

Reply:
Unfortunately not, since with symlinks the dotnet CLI does not find all specified dotnet versions - only the one the CLI belongs to.